src/Test/Functional/Acceptance/DatabaseConfigurationCest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Test\Functional\Acceptance;
+
+use Magento\MagentoCloud\Test\Functional\Codeception\Docker;
+
+/**
+ * This test runs on the latest version of PHP
+ */
+class DatabaseConfigurationCest extends AbstractCest
+{
+    /**
+     * @param \CliTester $I
+     * @throws \Robo\Exception\TaskException
+     */
+    public function _before(\CliTester $I)
+    {
+        parent::_before($I);
+        $I->cloneTemplate();
+        $I->addEceComposerRepo();
+    }
+
+    /**
+     * @param \CliTester $I
+     * @param \Codeception\Example $data
+     * @throws \Robo\Exception\TaskException
+     * @dataProvider databaseConfigurationDataProvider
+     */
+    public function databaseConfiguration(\CliTester $I, \Codeception\Example $data)
+    {
+        $I->assertTrue($I->runEceToolsCommand('build', Docker::BUILD_CONTAINER));
+        $I->startEnvironment();
+        $I->assertTrue($I->runEceToolsCommand(
+            'deploy',
+            Docker::DEPLOY_CONTAINER,
+            $data['cloudVariables']
+        ));
+        $file = $I->grabFileContent('/app/etc/env.php');
+        $I->assertContains($data['mergedConfig'], $file);
+        $I->assertContains($data['defaultConfig'], $file);
+    }
+
+    /**
+     * @return array
+     */
+    protected function databaseConfigurationDataProvider(): array
+    {
+        return [
+            '_merge1' => [
```

Comment:
Instead of `_merge1` and `_merge2` you can describe in a few words what test is checking.

Reply:
done